config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3736,6 +3736,10 @@
 # DEFAULT: false
 # samigo.settings.backgroundcolor.enabled=true
 
+# Ability to reshuffle all the parts using random questions from question pools
+# DEFAULT: false
+# amigo.all.random.parts.reshuffle.enabled=true
```

Comment:
First letter s is missing:

**s**amigo.all.random.parts.reshuffle.enabled=true

Reply:
I meant amigo, gracias amigo!